twistedchecker/checkers/python3.py

```diff
@@ -1,187 +0,0 @@
-"""
-Checker for python3 compatibility issues.
```

Comment:
By the way, there is also pylint --py3k which you might use for the same problem. It seems more feature complete than what you are currently using, maybe it is worth taking a look.